_sources/configs/composer.wikiteq.json

```diff
@@ -1,52 +1,48 @@
 {
 	"require": {
-		"elasticsearch/elasticsearch": "6.7.2",
-		"mediawiki/chameleon-skin": "^3.4.1",
-		"mediawiki/abuse-filter": "dev-master#188389ce51055b59f338a6784ecb51b30973e81f",
-		"mediawiki/maps": "8.0.0",
-		"mediawiki/semantic-media-wiki": "3.2.3",
-		"mediawiki/semantic-result-formats": "3.2.0",
-		"mediawiki/semantic-breadcrumb-links": "2.0",
-		"mediawiki/semantic-compound-queries": "2.1.0",
-		"mediawiki/semantic-extra-special-properties": "2.1.0",
-		"mediawiki/semantic-scribunto": "2.1.0",
-		"mediawiki/simple-batch-upload": "1.8.2",
-		"mediawiki/bootstrap-components": "4.0.1",
-		"mediawiki/sub-page-list": "1.6.1",
-		"mediawiki/remote-wiki": "dev-master#cb31c362e0759c95b0b97f03ad3b2b08c2e77529"
+		"mediawiki/semantic-media-wiki": "4.0.2",
```

Comment:
Accordingly to the https://www.semantic-mediawiki.org/wiki/Help:Compatibility we must use `4.1.x` release of SWM for `1.39`

Reply:
thanks, yeah, let's not try to stick close to Canasta, that's fine for Taqasta to evolve in an own way